prompt-api-playground/script.js

```diff
@@ -0,0 +1,98 @@
+import { marked } from 'https://cdn.jsdelivr.net/npm/marked@13.0.3/lib/marked.esm.js';
+import DOMPurify from 'https://cdn.jsdelivr.net/npm/dompurify@3.1.6/dist/purify.es.mjs';
+
+(async () => {
+  const errorMessage = document.getElementById('error-message');
+  const promptArea = document.getElementById('prompt-area');
+  const problematicArea = document.getElementById('problematic-area');
+  const promptInput = document.getElementById('prompt-input');
+  const responseArea = document.getElementById('response-area');
+  const copyLinkButton = document.getElementById('copy-link-button');
+  const copyHelper = document.querySelector('small');
+  const rawResponse = document.querySelector('details div');
+  const form = document.querySelector('form');
+  responseArea.style.display = 'none';
+
+  if (!window.ai) {
+    errorMessage.style.display = 'block';
+    errorMessage.innerHTML = `Your browser doesn't support the Prompt API. If you're on Chrome, join the <a href="https://docs.google.com/forms/d/e/1FAIpQLSfZXeiwj9KO9jMctffHPym88ln12xNWCrVkMY_u06WfSTulQg/viewform?resourcekey=0-dE0Rqy_GYXDEWSnU7Z0iHg">Early Preview Program</a> to enable it.`;
```

Comment:
This makes me think we need a landing page for the EPP on developer.chrome.com (We've been linking to https://developer.chrome.com/docs/ai/built-in)

Reply:
Link changed